astro-client/build-astro-clients.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/sh
+
+# This script is used to generate a golang client for the Astro REST public API
+# TODO: for now it gets the specs from the prod server, but eventually we will publish the specs in a public repo
```

Comment:
eventually we will release the entire clients as a public repo/package. so no need to generate clients anymore in cli